src/hooks/offramp/useMainProcess.ts

```diff
@@ -129,10 +129,6 @@ export const useMainProcess = () => {
     setSigningPhase,
   ]);
 
-  useEffect(() => {
```

Comment:
As per the previous comment, just setting `setOnSelectedNetworkChange(resetOfframpingState);` causes it to trigger on mount, which removes a potential state that must be picked up for continuation.

Since the network selector is disabled during the off-ramp, I think we can remove this now.

If we still want to add it back in the future, a potential solution exists in [this](https://github.com/pendulum-chain/vortex/pull/323/files#diff-bb2814ac946cf36194efa0ba6b097de7881a6780c27ef1412100165ece04ce09R89) WIP PR, possible after merging [this](https://github.com/pendulum-chain/vortex/pull/324/files#diff-d2caeb4570ef1c2b7edf7b74ad89218b6f82a0a640967070b48a343766e18c66).